### PR TITLE
Ek 149 replace gatsby source muli api with gatsby source custom api

### PR DESCRIPTION
**Pull request by dmotan**
In this PR, I have changed FP to use `gatsby-source-custom-api` instead of `gatsby-source-multi-api` since the latter has been archived and no longer supported.

### CODE REVIEW

File: gatsby_fp/gatsby-config.js [hunk 1]
```diff
@@ -7,11 +7,87 @@ module.exports = {
...
+            room: String
+            photo_base64: String
+            notable_courses: String
+            school: String
+            ses_department: String
+            service_university: [service_university]
+            service_professional: [service_professional]
+            education: [education]
+            member: [member]
+            intellcont: [intellcont]
```
[the remark this answers is not in the document]
Comment (dmotan) [reply gatsby_fp/gatsby-config.js:44]: Can you give an example, please?

[the remark this answers is not in the document]
Comment (cs-paz) [reply gatsby_fp/gatsby-config.js:44]: Of course @dmotan 
Upon build for, intellcont auth, we get the message 
`warn Deprecation warning - adding inferred resolver for field profiles.education`
By removing 
`education: [education]`
This message is resolved and everything works as it should, and this is the same for every other array in the schema

[the remark this answers is not in the document]
Comment (dmotan) [reply gatsby_fp/gatsby-config.js:44]: How could this happen? How gatsby is going to know that `intellcont_auth` belongs to `intellcont`?

[the remark this answers is not in the document]
Comment (cs-paz) [reply gatsby_fp/gatsby-config.js:44]: Right... that is correct, didn't even notice that difference when looking at a local build.  I think I was incorrect by saying we should delete the inferred resolvers, but maybe we should add some type of extension?  See this article here 
https://www.gatsbyjs.org/blog/2019-05-17-improvements-to-schema-customization/#resolver-extensions

[the remark this answers is not in the document]
Comment (mf-stevens) [reply gatsby_fp/gatsby-config.js:44]: [If this is a good example](https://github.com/panr/gatsby-starter-hello-friend/pull/10) (and I'm not sure that it is, but it was the first result when I googled the warning) "adding inferred resolver" means that you were supposed to have an explicit resolver, but you forgot. The fix is to make it explicit, not to omit the key from the schema.

[the remark this answers is not in the document]
Comment (mf-stevens) [reply gatsby_fp/gatsby-config.js:44]: [Here's a better explanation](https://www.jamesdflynn.com/development/gatsbyjs-drupal-create-custom-graphql-schema-empty-fields) -- specifically the part that says "If you don't include the @link ... in your query you'll get a deprecation notice"

[the remark this answers is not in the document]
Comment (dmotan) [reply gatsby_fp/gatsby-config.js:44]: I still don't get it, if someone understands and can implement what all of the examples you guys provided, please feel free to paste a snippet here! All of them are talking about how to fix this in grapql not in the schema.

[the remark this answers is not in the document]
Comment (mf-stevens) [reply gatsby_fp/gatsby-config.js:44]: I'm not sure either. Let's see if @davidwatson can provide an example of what in our code would need to change. If not:
We're blocked from upgrading from gatsbyjs v2 to v3, but given that the release schedule doesn't even mention v3 yet, I assume there is _plenty_ of time to sort it out, in which case I'm fine with the warnings.

[the remark this answers is not in the document]
Comment (mf-stevens) [reply gatsby_fp/gatsby-config.js:44]: We'll go ahead and proceed, accepting the warnings as something to solve before upgrading to Gatsby v3.

File: gatsby_fp/src/components/Layout.js [hunk 2]
```diff
@@ -33,12 +33,12 @@ export default function Layout({ children, data }) {
     <>
       {data
         ? <Head
-            pageTitle={`${sanitizedData.pf_first_name} ${sanitizedData.pf_last_name} - Stevens Institute of Technology`}
```
Comment (mf-stevens) [inline gatsby_fp/src/components/Layout.js:36]: Why are we removing this indentation?

File: gatsby_fp/gatsby-config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidwatson) [reply gatsby_fp/gatsby-config.js:44]: Looping back to this, I don't have a very good solution off-hand. Given that we have other things that need prioritizing, and that it doesn't appear to disrupt anything else, I'd say this is debt we can accrue for now.

**Pull request merged**